conan/internal/graph/graph_binaries.py

```diff
@@ -211,6 +211,7 @@ def _compatible_find_existing_binaries(self, node, compatibles, remotes, update)
                     self._compatible_found(conanfile, package_id, compatible_package)
                     return
 
+        node.conanfile.output.info("No compatible configuration not found", fg=Color.BRIGHT_CYAN)
```

Comment:
```suggestion
        node.conanfile.output.info("No compatible configuration found", fg=Color.BRIGHT_CYAN)
```
Or maybe "Compatible configuration not found"?

Reply:
Oops, good catch